src/System.Management.Automation/engine/ExecutionContext.cs

```diff
@@ -1258,14 +1261,20 @@ internal static Assembly LoadAssembly(string name, string filename, out Exceptio
                 catch (BadImageFormatException badImage)
```

Comment:
The comments in the `catch (FileLoadException fileLoadException)` block right above this line seems not needed anymore, since we now try loading from path first.

Reply:
resolved